app/src/main/res/layout/activity_reminder_edit.xml

```diff
@@ -25,16 +41,17 @@
             android:text="@string/button_create_new"
             android:layout_alignParentBottom="true"
             android:layout_alignParentRight="true"
+            android:layout_alignParentEnd="true"
```

Comment:
`alignParentEnd` was added in API 17. To support older versions, you include both. I believe API 17+ will see both and ignore the older one (`right`).